### ISSUE TITLE
Need bs4Dash >= 0.6.0.900 but loaded version is 0.5.0 

### ISSUE THREAD

**Issue by mbacou** [RinteRface/bs4Dash#143]
Hi,
I tried to update/test **bs4Dash v2.0.0** but failed to install dependencies so had to revert to an earlier release (my previous installation was v0.6.0900). v0.5 is not compatible with my app. How can I re-install **v0.6** ? I'm a bit puzzled why this release is no longer showing in this repo.
Thanks!

**Comment by mbacou** [comment]
Never mind, that worked for me:
```
devtools::install_github("RinteRface/bs4Dash@2f3aedec89c38e2dd876f02f3cb11cb9c1745eac")
```

**Comment by DivadNojnarg** [comment]
Sure but I would have been curious to see the installation error. Could you show it to me?

**Comment by mbacou** [comment]
@DivadNojnarg I think it was permissioning issue on my end, as I successfully installed on a local machine. Looks like on our server `rlang` could not be unloaded and updated, and that failed the rest of the process.
Is there any breaking change between **v0.6.0** and **v2.0.0**? If not, I can give it another try and report back.

```
> devtools::install_github("RinteRface/bs4Dash")
Downloading GitHub repo RinteRface/bs4Dash@HEAD
These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                                        
2: CRAN packages only                         
3: None                                       
4: htmltools  (0.5.0 -> 636b95eaf...) [GitHub]
5: shiny      (1.5.0 -> 11babd556...) [GitHub]
6: magrittr   (1.5   -> 2.0.1       ) [CRAN]  
7: rlang      (0.4.8 -> 0.4.9       ) [CRAN]  
8: rstudioapi (0.11  -> 0.13        ) [CRAN]  

Enter one or more numbers, or an empty line to skip updates:
```
Then:
```
Downloading GitHub repo rstudio/shiny@HEAD
These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                                      
2: CRAN packages only                       
3: None                                     
4: promises (1.1.1 -> bbadb3d7d...) [GitHub]
5: rlang    (0.4.9 -> 54e2cfa28...) [GitHub]

Enter one or more numbers, or an empty line to skip updates:
```
Then
```
Downloading GitHub repo rstudio/bslib@HEAD
These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                                  
2: CRAN packages only                   
3: None                                 
4: sass (0.2.0 -> fef1d255c...) [GitHub]

Enter one or more numbers, or an empty line to skip updates:1
```

**Comment by DivadNojnarg** [comment]
> Is there any breaking change between v0.6.0 and v2.0.0? If not, I can give it another try and report back

Yes there are a lot. Unfortunately, v0.6.0 was based on an unstable release of AdminLTE with too many issues. v2.0.0 provides a direct 1:1 support with shinydashboard and shinydashboardPlus, as well as many bug fixes.

**Comment by mbacou** [comment]
Good existing news! But in that case I won't be able to test right away, need a few days to plan an upgrade.

**Comment by mbacou** [comment]
@DivadNojnarg I managed to successful install on our VM now by skipping the recommended updates. So false alarm I'd say. 

Upgrading our apps was a no-brainer (just a few hiccups when using a vertical menu in the right controlbar, see below). Thanks for this excellent work!

![image](https://user-images.githubusercontent.com/260325/100467701-d9c49c00-30ca-11eb-8a9e-e13f172874b3.png)

Fixed by adding the following `<div>` below: 
![image](https://user-images.githubusercontent.com/260325/100467783-02e52c80-30cb-11eb-928d-ac14dff45637.png)

![image](https://user-images.githubusercontent.com/260325/100468152-c0701f80-30cb-11eb-8223-b3abb1b1b8bc.png)
